tests/integration/cartography/intel/aws/test_rds_event_subscriptions.py

```diff
@@ -0,0 +1,247 @@
+import cartography.intel.aws.rds
+from tests.data.aws.rds import DESCRIBE_DBCLUSTERS_RESPONSE
+from tests.data.aws.rds import DESCRIBE_DBINSTANCES_RESPONSE
+from tests.data.aws.rds import DESCRIBE_DBSNAPSHOTS_RESPONSE
+from tests.data.aws.rds import DESCRIBE_EVENT_SUBSCRIPTIONS_RESPONSE
+from tests.data.aws.sns import TEST_RDS_EVENT_SUBSCRIPTION_TOPICS
+from tests.integration.cartography.intel.aws.common import create_test_account
+from tests.integration.util import check_nodes
+from tests.integration.util import check_rels
+
+TEST_ACCOUNT_ID = "000000000000"
+TEST_REGION = "us-east-1"
+TEST_UPDATE_TAG = 123456789
+
+
```

Comment:
Please change the tests to simply cover your sync function. If you're using an agent, point it at https://github.com/cartography-cncf/cartography/blob/master/AGENTS.md#integration-tests. 

Do the arrange+act+assert pattern, patch out the get function, call the sync in the act phase, then the main thing we want to assert is that the expected relationships exist.